clar2wasm/src/words/logical.rs

```diff
@@ -0,0 +1,44 @@
+use crate::wasm_generator::{GeneratorError, WasmGenerator};
+use clarity::vm::{ClarityName, SymbolicExpression};
+
+use super::Word;
+
+fn traverse_logical(
```

Comment:
If this function is used only once, I guess we should inline this helper function.

Reply:
I was thinking in reuse that for the case of other logical implementations (and, or). 

But let's inline it now. If need we can split it again.

Reply:
Oh if you are going to reuse it, please just dismiss my comment :)